pyvlx/pyvlx.py

```diff
@@ -24,28 +24,36 @@
 class PyVLX:
     """Class for PyVLX."""
 
-    def __init__(self,
-                 path: Optional[str] = None,
-                 host: Optional[str] = None,
-                 password: Optional[str] = None,
-                 loop: Optional[asyncio.AbstractEventLoop] = None):
+    def __init__(
+        self,
+        path: Optional[str] = None,
+        host: Optional[str] = None,
+        password: Optional[str] = None,
+        loop: Optional[asyncio.AbstractEventLoop] = None,
+        heartbeat_interval: int = 30,
+        heartbeat_load_all_states: bool = True,
+    ):
         """Initialize PyVLX class."""
         self.loop = loop or asyncio.get_event_loop()
         self.config = Config(self, path, host, password)
         self.connection = Connection(loop=self.loop, config=self.config)
-        self.heartbeat = Heartbeat(pyvlx=self)
+        self.heartbeat = Heartbeat(
+            pyvlx=self,
+            interval=heartbeat_interval,
+            load_all_states=heartbeat_load_all_states,
+        )
         self.node_updater = NodeUpdater(pyvlx=self)
         self.connection.register_frame_received_cb(self.node_updater.process_frame)
         self.nodes = Nodes(self)
         self.scenes = Scenes(self)
         self.version = None
         self.protocol_version = None
         self.klf200 = Klf200Gateway(pyvlx=self)
+        self.semaphore = asyncio.Semaphore(1)  # Limit parallel commands
```

Comment:
Please change to: `self.api_call_semaphore = asyncio.Semaphore(1)  # Limit parallel commands` - so it wont conflict.